.github/workflows/deploy-docs.yaml

```
@@ -5,12 +5,16 @@ on:
     branches:
       - main
       - galactic
+      - beta/*
     paths:
       - mkdocs.yaml
       - "**/*.md"
       - "**/*.svg"
       - "**/*.png"
       - "**/*.jpg"
+    tags:
+      - v*
+      - "[0-9]+.[0-9]+*"
```

```suggestion
      - "[0-9]*.[0-9]*.[0-9]*"
```

I suggest just versioning with semver, not allowing just `1.2` for example. Only `1.2.0` 3 sections allowed.
Also let's not allow `v1.2` in tags. Only direct numbers. So let's rename the existing `v1.0` tag to `1.0.0` is that alright?

---

https://github.com/xmfcx/autoware-documentation/actions tested this new `"[0-9]*.[0-9]*.[0-9]*"` regex in my local fork. It works with 3 number tag push.

---

- https://github.com/autowarefoundation/autoware-github-actions/pull/362

Here added support for pushing with the tag name. The tests are included in that PR.